render.js

```diff
@@ -0,0 +1,42 @@
+const Renderer = function() {
+   
+  const renderPosts = function(posts) { 
```

Comment:
Overall good, but you should probably separate the different concerns to separate functions. I would have at least 3 functions here, for example:
1. To create post
2. To create comment
3. To manage them both and append.